### PR TITLE
Duplicated 'in' claim from 'bug' label

### PR DESCRIPTION
**Pull request by g4s8**
#391 - check WBS in RbOnBug

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#727](https://github.com/zerocracy/farm/issues/727) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#727](https://github.com/zerocracy/farm/issues/727) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/641354299). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/java/com/zerocracy/radars/github/RbOnBug.java [hunk 1]
```diff
@@ -40,15 +42,25 @@ public String react(final Farm farm, final Github github,
...
-            .type("Add job to WBS")
-            .token(new TokenOfIssue(issue))
-            .param("job", new Job(issue))
-            .param("reason", "GitHub label was attached")
-            .postTo(new GhProject(farm, issue.repo()));
-        return new FormattedText(
-            "Issue #%d added to WBS by 'bug' label",
-            issue.number()
-        ).asString();
+        final GhProject project = new GhProject(farm, issue.repo());
```
Comment (yegor256) [inline src/main/java/com/zerocracy/radars/github/RbOnBug.java]: @g4s8 it's better to move this check into the Stakeholder, see: 1a8acd3a

**Comment by yegor256** [comment]
@g4s8 see above

**Comment by g4s8** [comment]
@yegor256 done - moved to stakeholder

File: src/main/java/com/zerocracy/radars/github/RbOnBug.java [hunk 2]
```diff
@@ -45,6 +45,7 @@ public String react(final Farm farm, final Github github,
             .token(new TokenOfIssue(issue))
             .param("job", new Job(issue))
             .param("reason", "GitHub label was attached")
+            .param("label", "bug")
```
Comment (yegor256) [inline src/main/java/com/zerocracy/radars/github/RbOnBug.java]: @g4s8 I would better call it `.param("quiet", true)`

File: src/main/resources/com/zerocracy/stk/pm/scope/wbs/add_job_to_wbs.groovy [hunk 3]
```diff
@@ -34,12 +34,17 @@ def exec(Project project, XML xml) {
   new Assume(project, xml).type('Add job to WBS')
   new Assume(project, xml).roles('ARC', 'PO')
   ClaimIn claim = new ClaimIn(xml)
+  Wbs wbs = new Wbs(project).bootstrap()
   String job = claim.param('job')
+  if (claim.hasParam('label') && claim.param('label') == 'bug') {
```
Comment (yegor256) [inline src/main/resources/com/zerocracy/stk/pm/scope/wbs/add_job_to_wbs.groovy]: @g4s8 let's use `quiet` or maybe even `ignore-duplicates`

**Comment by yegor256** [comment]
@g4s8 see above

**Comment by g4s8** [comment]
@yegor256 fixed in 5b28d5778fad015a2e61fc9fef71b30c81cde3c3

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
The issue is closed not by its creator, I won't close the order; please, re-open it and ask its creator to close it

**Comment by yegor256** [comment]
@0crat out

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by ypshenychka** [comment]
@carlosmiranda I see that there was no CR done from your side at all. Could you please explain why?

**Comment by carlosmiranda** [comment]
@ypshenychka The review was started by the architect (@yegor256) before I had a chance to propose any changes to this ticket. This was not by choice.

**Comment by ypshenychka** [comment]
@carlosmiranda I see. Well, I can't give any positive verdict here since no work was actually done by code reviewer. Hope you understand.

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
`QA` review completed: +8 points just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by carlosmiranda** [comment]
@ypshenychka no problem. :)

**Pull request merged**